newIDE/app/src/MainFrame/EditorContainers/HomePage/CardWidget.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import * as React from 'react';
+import { createStyles, makeStyles, Paper } from '@material-ui/core';
+import { shouldValidate } from '../../../UI/KeyboardShortcuts/InteractionKeys';
+import { useResponsiveWindowWidth } from '../../../UI/Reponsive/ResponsiveWindowMeasurer';
+
+const styles = {
+  paper: {
+    borderRadius: 8,
+    cursor: 'default',
+    height: '95%', // Take as much space as possible, without hiding the bottom border.
+  },
+};
+
+// Styles to give the impression of pressing an element.
+const useStylesForWidget = makeStyles(theme =>
+  createStyles({
+    root: {
+      border: `1px solid ${theme.palette.text.primary}`,
+      borderBottom: `6px solid ${theme.palette.text.primary}`,
+      '&:focus': {
```

Comment:
I don't know if there is something to do on :focus-visible to avoid:

<img width="200" alt="image" src="https://user-images.githubusercontent.com/32449369/177335417-b57fee1b-7b66-46d3-80eb-34f62d8d177b.png">

Reply:
yep, we're reworking this design altogether